### DISCUSSION TITLE
Docker Push GHCR error

### DISCUSSION THREAD

**Discussion by Nollknolle** [community/community#26255] [Actions]
<p>Hey when i push my image to <a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a> the i get a weird error.</p>

**Comment by Nollknolle** [comment]
<pre><code class="lang-auto">#11 [auth] nollknolle/chipx9:pull,push token for ghcr.io
#11 DONE 0.0s

#10 exporting to image
#10 pushing layers 5.4s done
#10 pushing manifest for ghcr.io/nollknolle/chipx9:latest@sha256:1299621dccfe32b6adc6d0f5411d6cf0c86d95926e12079a16a8e32e8edf2405
#10 pushing manifest for ghcr.io/nollknolle/chipx9:latest@sha256:1299621dccfe32b6adc6d0f5411d6cf0c86d95926e12079a16a8e32e8edf2405 0.1s done
#10 ERROR: unexpected status: 499 status code 499
------
 &gt; exporting to image:
------
error: failed to solve: unexpected status: 499 status code 499
Error: buildx failed with: error: failed to solve: unexpected status: 499 status code 499
</code></pre>
<pre><code class="lang-auto">name: ci
on:
  push:
    branches:
      - "main"

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout 
        uses: actions/checkout@v2
      - name: Login to Docker Hub
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: nollknolle
          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1
      - name: Build and push
        uses: docker/build-push-action@v2
        with:
          context: .
          file: ./Dockerfile
          push: true
          tags: ghcr.io/nollknolle/chipx9:latest
</code></pre>

**Comment by Zenexer** [comment]
<p>You’re not the only one: <a href="https://github.community/t/suddenly-getting-499-from-github-container-registry/261463" class="inline-onebox">Suddenly getting 499 from GitHub container registry</a></p>

**Comment by Nollknolle** [comment, marked as answer]
<aside class="quote" data-post="7" data-topic="261463">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/suddenly-getting-499-from-github-container-registry/261463/7">Suddenly getting 499 from GitHub container registry</a> <a class="badge-wrapper  bullet" href="/c/code-to-cloud/github-actions/41"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #2188ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Actions—automation for all of your development workflows.">GitHub Actions</span></a>
  </div>
  <blockquote>
    This issue seemed to be resolved for me, I’m not getting any errors anymore.
  </blockquote>
</aside>
